crux-bench/bin/run_task.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 aws ecs run-task\
- --task-definition crux-bench\
+ --task-definition crux-bench-latest\
```

Comment:
rather than having multiple task definitions per version, could we specify this as a container override too?

Reply:
turns out it's not possible to override the image version. shame!